### PR TITLE
Create a 'Deploy to Heroku' Button

### PR DESCRIPTION
**Pull request by Saptarshee94**
This Pull request adds the `Deploy to Heroku` button in the Readme.md  of the project.

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        "SMTP_FROM": {
+            "description": "SMTP server sender's email address."
+        },
+        "DATABASE_URL": {
+            "description": "Hosted Database URL."
+        },
+        "APP_URL": {
+            "description": "Public root URL of the Enterprise SaaS installation, replace <HEROKU_APP_NAME> with the app name.",
+            "value": "https://<HEROKU_APP_NAME>.herokuapp.com"
+        },
```
Comment (devkiran) [inline app.json:40]: @Saptarshee94 Let's make the SVIX_URL, and SVIX_API_KEY as **optional**. These are required only if the app use the Webhook feature. The developer can add these value later from the Heroku env page.

**Comment by devkiran** [comment]
I tried the Deploy button, but got the following error. I think you have to update the PORT on the package.json

```
2022-10-05T06:15:17.120528+00:00 heroku[web.1]: Error R10 (Boot timeout) -> Web process failed to bind to $PORT within 60 seconds of launch
2022-10-05T06:15:17.216885+00:00 heroku[web.1]: Stopping process with SIGKILL
2022-10-05T06:15:17.369567+00:00 heroku[web.1]: Process exited with status 137
2022-10-05T06:15:17.550423+00:00 heroku[web.1]: State changed from starting to crashed
```

File: app.json (hunk 1 above)
Comment (Saptarshee94) [reply app.json:40]: Resolved issue by making `SVIX_URL` and `SVIX_API_KEY `  optional.

**Comment by Saptarshee94** [comment]
> I tried the Deploy button, but got the following error. I think you have to update the PORT on the package.json
> 
> ```
> 2022-10-05T06:15:17.120528+00:00 heroku[web.1]: Error R10 (Boot timeout) -> Web process failed to bind to $PORT within 60 seconds of launch
> 2022-10-05T06:15:17.216885+00:00 heroku[web.1]: Stopping process with SIGKILL
> 2022-10-05T06:15:17.369567+00:00 heroku[web.1]: Process exited with status 137
> 2022-10-05T06:15:17.550423+00:00 heroku[web.1]: State changed from starting to crashed
> ```

Issue resolved by removing --port option from `npm start` command.

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "dev": "next dev --port 4002",
     "build": "next build",
-    "start": "next start --port 4002",
```
[the remark this answers is not in the document]
Comment (Saptarshee94) [reply package.json]: Changes reverted and added `Procfile` to resolve the issue.

File: Procfile [hunk 3]
```diff
@@ -0,0 +1 @@
+web: ./node_modules/.bin/next start
```
Comment (devkiran) [inline Procfile:1]: Does this work without a $PORT value?

Comment (Saptarshee94) [reply Procfile:1]: Yes, I have deployed it on a heroku server to test with the `procfile`, It successfully build and deployed without any `$PORT` 
related issue. All the issue I found was with the '.env' vars as I provided some random data for it. Is it neccessary to put that `$PORT` value for deployment?  
image

**Comment by devkiran** [comment]
Looks good to me.

**Review by devkiran — APPROVED** [review]

**Comment by devkiran** [comment]
Resolved #32

**Pull request merged**